From 343fa95f20146aa07d8e8ca8fa1447bc3d42b482 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Thu, 25 Apr 2024 01:17:33 -0700
Subject: [PATCH 4/4] fix(cups-pdf.conf): output PDFs to a more convenient
 location

Follow Debian and output all "printed" PDFs to $HOME/PDF.
---
 extra/cups-pdf.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/extra/cups-pdf.conf b/extra/cups-pdf.conf
index eb568c7..c7dc3d2 100644
--- a/extra/cups-pdf.conf
+++ b/extra/cups-pdf.conf
@@ -48,7 +48,7 @@
 ##  root_squash! 
 ### Default: /var/spool/cups-pdf/${USER}
 
-#Out /var/spool/cups-pdf/${USER}
+Out ${HOME}/PDF
 
 ### Key: AnonDirName (config)
 ##  ABSOLUTE path for anonymously created PDF files
-- 
2.44.0

